tests/test_tokenization_canine.py

```diff
@@ -39,11 +39,12 @@ def setUp(self):
 
     @cached_property
     def canine_tokenizer(self):
-        # TODO replace nielsr by google
-        return CanineTokenizer.from_pretrained("nielsr/canine-s")
+        return CanineTokenizer.from_pretrained("google/canine-s")
 
     def get_tokenizer(self, **kwargs) -> CanineTokenizer:
-        return self.tokenizer_class.from_pretrained(self.tmpdirname, **kwargs)
+        tokenizer = self.tokenizer_class.from_pretrained(self.tmpdirname, **kwargs)
+        tokenizer._unicode_vocab_size = 1024
```

Comment:
This change alone gives a speed up of 2.5 minutes. The reason is that `len(tokenizer)` is by default > 1,000,000 for every Canine tokenizer and the 4 slowest tests encode every single character in the vocabulary in a for loop.

Reply:
Nice catch!